app/models/user.py

```diff
@@ -0,0 +1,43 @@
+from app.controllers import user
+
+class User:
```

Comment:
This would be better done with SQLAlchemy, for proper, simpler ORM.

Reply:
ok, googling that, it does seem like it is what this should be. Should I just remove this class from this PR then?

Reply:
Sure